build_packages.sh

```diff
@@ -48,10 +56,47 @@ function build_distro() {
     setup.py
 }
 
-# RHEL/CentOS
-build_distro 'el6' 'rpm' '/usr/lib/python2.6/site-packages'
-build_distro 'el7' 'rpm' '/usr/lib/python2.7/site-packages'
+TIMESTAMP="$(date +%s)"
+
+while getopts 'ho:' OPTION; do
+  case "$OPTION" in
+    h)
+      usage
+      exit 2
+      ;;
+    o)
+      set -f
+      IFS=','
+      BUILD=($OPTARG)
+      set +f
+      ;;
+    ?)
+      usage
+      exit
+      ;;
+  esac
+done
+
+if [ -z "$BUILD" ]; then
+  BUILD=('el6' 'el7' 'wheezy' 'jessie')
+fi
 
-# Debian
-build_distro 'wheezy' 'deb' '/usr/lib/python2.7/dist-packages'
-build_distro 'jessie' 'deb' '/usr/lib/python2.7/dist-packages'
+for build in "${BUILD[@]}"; do
+  case "$build" in
+  el6) # RHEL/CentOS 6
```

Comment:
Just noticed this - please indent alternatives by 2 spaces (like you did above).